### PR TITLE
feat: Add the 'cert' argument to the Requests framework

### PR DESCRIPTION
**Pull request by tomytrip**
To access the Metabase database in my company, we need to present a certificate to prove our identity. 
To do this, we need to use the Requests argument: "cert"
So I add it in the MetabaseClient object

### CODE REVIEW

**Review by kozlek — APPROVED** [review]

File: dbtmetabase/metabase.py [hunk 1]
```diff
@@ -144,11 +147,11 @@ def __init__(
         self.base_url = f"{'http' if use_http else 'https'}://{host}"
         self.session = requests.Session()
         self.session.verify = verify
+        self.session.cert = cert
         adaptor = HTTPAdapter(max_retries=Retry(total=3, backoff_factor=0.5))
         self.session.mount(self.base_url, adaptor)
         session_header = session_id or self.get_session_id(user, password)
         self.session.headers["X-Metabase-Session"] = session_header
-
```
Comment (gouline) [inline dbtmetabase/metabase.py:151]: Unrelated change, please revert

File: dbtmetabase/metabase.py [hunk 2]
```diff
@@ -120,6 +121,7 @@ def __init__(
         user: str,
         password: str,
         verify: Optional[Union[str, bool]] = None,
+        cert: Optional[Union[str, Tuple[str, str]]] = None,
```
Comment (gouline) [inline dbtmetabase/metabase.py:124]: So it's either a string or a tuple of two strings?

Comment (tomytrip) [reply dbtmetabase/metabase.py:124]: Yes, you can verify it in the requests documentation at the Client Side Certificates part https://requests.readthedocs.io/en/latest/user/advanced/

**Review by gouline — APPROVED** [review]
